web-app/src/components/common_styled/Ul.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
```

Comment:
This whole file shouldn't exist I think. You've got it in common_styled but there are very specific classes in here, which takes out of "common" territory.